libexec/bats-exec-test

```diff
@@ -263,6 +283,12 @@ bats_exit_trap() {
     status=0
   fi
 
+  if [[ -z $BATS_EXTENDED_SYNTAX && -n $BATS_RECORD_DURATIONS && -n $BATS_TEST_DURATION ]]; then
```

Comment:
Why not just show this inline instead? Something like extending `$BATS_TEST_DESCRIPTION` with `(duration ms: BATS_TEST_DURATION)`.